File: src/test/java/net/slipp/dto/QuestionDTO.java
package net.slipp.dto;

import com.fasterxml.jackson.annotation.JsonIgnoreProperties;

@JsonIgnoreProperties(ignoreUnknown = true)
public class QuestionDTO {
	private String title;
	private String contents;

	public String getTitle() {
		return title;
	}

	public QuestionDTO setTitle(String title) {
		this.title = title;
		return this;
	}

	public String getContents() {
		return contents;
	}

	public QuestionDTO setContents(String contents) {
		this.contents = contents;
		return this;
	}
}

File: src/main/kotlin/net/slipp/dto/QuestionDTO.kt
package net.slipp.dto

import com.fasterxml.jackson.annotation.JsonIgnoreProperties

@JsonIgnoreProperties(ignoreUnknown = true)
data class QuestionDTO(var id: Long, var title: String, var contents: String) {
    constructor(): this(0, "", "")

    constructor(title: String, contents: String): this(0, title, contents)
}
